packages/tools/src/workers/computeWorker.ts

```diff
@@ -0,0 +1,132 @@
+import { expose } from 'comlink';
+import VolumetricCalculator from '../utilities/segmentation/VolumetricCalculator';
+import { peerImport } from '@cornerstonejs/core';
+import getItkImage from '../tools/segmentation/strategies/utils/getItkImage';
+import vtkImageData from '@kitware/vtk.js/Common/DataModel/ImageData';
+import vtkDataArray from '@kitware/vtk.js/Common/Core/DataArray';
+
+const computeWorker = {
+  calculateSegmentsStatistics: (args) => {
+    const { segmentationInfo, imageInfo, indices } = args;
+
+    const {
+      scalarData: segmentationScalarData,
+      dimensions: segmentationDimensions,
+      spacing: segmentationSpacing,
+    } = segmentationInfo;
+    const { scalarData: imageScalarData, dimensions: imageDimensions } =
+      imageInfo;
+
+    // if dimensions are not the same, for now just throw an error
+    if (
+      segmentationDimensions[0] !== imageDimensions[0] ||
+      segmentationDimensions[1] !== imageDimensions[1] ||
+      segmentationDimensions[2] !== imageDimensions[2]
+    ) {
+      throw new Error('Dimensions do not match');
+    }
+
+    for (let i = 0; i < segmentationScalarData.length; i++) {
+      const segmentationValue = segmentationScalarData[i];
+
+      if (indices.indexOf(segmentationValue) === -1) {
+        continue;
+      }
+      const imageValue = imageScalarData[i];
+
+      VolumetricCalculator.statsCallback({
+        value: imageValue,
+        pointIJK: [
+          i % segmentationDimensions[0],
+          Math.floor(i / segmentationDimensions[0]) % segmentationDimensions[1],
+          Math.floor(i / segmentationDimensions[0] / segmentationDimensions[1]),
+        ],
+      });
+    }
+
+    const stats = VolumetricCalculator.getStatistics({
+      spacing: segmentationSpacing,
+      unit: 'mm',
+    });
+
+    return stats;
+  },
+
+  interpolateLabelmap: async (args) => {
+    const { segmentationInfo, configuration } = args;
+    const { scalarData, dimensions, spacing, origin, direction } =
+      segmentationInfo;
+
+    let itkModule;
+    try {
+      itkModule = await import('@itk-wasm/morphological-contour-interpolation');
```

Comment:
This will cause webpack/ohif to do a hard link to itk-wasm, breaking the fact that it should be able to load it dynamically.

Reply:
Done